modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java

```diff
@@ -2507,6 +2507,10 @@ private void _testUnbindObjectDefinitionsWithObjectAction(
 
 		Assert.assertFalse(objectAction.isActive());
 
+		Assert.assertEquals(
+			objectAction.getObjectActionTriggerKey(),
+			ObjectActionTriggerConstants.KEY_ON_AFTER_UPDATE);
```

Comment:
Can you sort and remove the empty lines between the Asserts?

```
Assert.assertEquals(
	objectAction.getObjectActionTriggerKey(),
	ObjectActionTriggerConstants.KEY_ON_AFTER_UPDATE);
Assert.assertFalse(objectAction.isActive());
```

Reply:
Sure